funnel/extapi/explara.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+
+import requests
+from ..util import format_twitter
+from ..models import SyncTicket
+
+__all__ = ['ExplaraAPI']
+
+
+class ExplaraAPI(object):
+    def __init__(self, access_token):
+        self.access_token = access_token
+        self.headers = {'Authorization': u'Bearer ' + self.access_token}
+        self.base_url = 'https://www.explara.com/api/e/{0}'
+
+    def get_events(self):
+        events = requests.post(self.base_url.format('get-all-events'), headers=self.headers).json()
+        return [{'title': e.get('eventTitle'), 'eventId': e.get('eventId')} for e in events.get('events')]
+
+    def get_ticket_types(self, explara_eventid):
+        return requests.post(self.base_url.format('get-tickets'), headers=self.headers, data={'eventId': explara_eventid}).json()
```

Comment:
Copy of the data from the API as needed, like in `get_events` instead of passing the raw Explara API response, which might be subject to change.